frontend/stylesheets/flex_layout_overwrites.less

```diff
@@ -121,3 +121,14 @@
 .hide_first_splitter_border ~ .hide_first_splitter_border {
   pointer-events: all;
 }
+
+.flex-container {
+  justify-content: space-between;
+  display: flex;
+}
+
+.flex-item {
+  display: inline-block;
+  flex-grow: 1;
+  word-break: break-all;
+}
```

Comment:
This file is actually meant to tune the FlexLayout library (has nothing to do with flexboxes, even though I see why it could seem like it :)). I'd suggest to move it into `_utils.less` as these are generic helper classes.